tests/test_docker.py

```diff
@@ -907,6 +911,65 @@ def post(req, resp):
 
 
 @if_docker
+def test_instance_activate_device_options(agent, responses):
+    delete_container('/c861f990-4472-4fa1-960f-65171b544c28')
+    # Note, can't test weight as it isn't supported in kernel by default
+    device_options = {'/dev/sda': {
+        'readIops': 1000,
+        'writeIops': 2000,
+        'readBps': 1024,
+        'writeBps': 2048
+        }
```

Comment:
Can you have a test case that options only has writeIops ? That ensure most user cases work, bc most of times, people won't put all options here.